### ISSUE TITLE
<<< FAILURE! - in com.jcabi.matchers.NoBrokenLinksTest

### ISSUE THREAD

**Issue by simonjenga** [jcabi/jcabi-matchers#27]
```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running com.jcabi.matchers.RegexMatchersTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.05 sec - in com.jcabi.matchers.RegexMatchersTest
Running com.jcabi.matchers.XhtmlMatchersTest
Tests run: 12, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.47 sec - in com.jcabi.matchers.XhtmlMatchersTest
Running com.jcabi.matchers.JaxbConverterTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.26 sec - in com.jcabi.matchers.JaxbConverterTest
Running com.jcabi.matchers.W3CMatchersTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.83 sec - in com.jcabi.matchers.W3CMatchersTest
Running com.jcabi.matchers.StringSourceTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec - in com.jcabi.matchers.StringSourceTest
Running com.jcabi.matchers.NoBrokenLinksTest
[?[2;37mDEBUG?[m] pool-1-thread-16 com.jcabi.matchers.NoBrokenLinks: #assertThat(): 2 links found: ["/index.html", "http://img.rexsl.com/logo.png"]
[?[2;37mDEBUG?[m] pool-1-thread-16 com.jcabi.matchers.NoBrokenLinks: #http('http://www.rexsl.com/index.html'): response code is 404
[?[0;33mWARN?[m] pool-1-thread-16 com.jcabi.matchers.NoBrokenLinks: #isValid('http://www.rexsl.com/index.html'): not valid since response code is 404
[?[0;33mWARN?[m] pool-1-thread-16 com.jcabi.matchers.NoBrokenLinks: #http('http://img.rexsl.com/logo.png'): can't get response code: img.rexsl.com
[?[0;33mWARN?[m] pool-1-thread-16 com.jcabi.matchers.NoBrokenLinks: #isValid('http://img.rexsl.com/logo.png'): not valid since response code is 400
[?[0;31mERROR?[m] pool-1-thread-10 com.jcabi.aspects.aj.MethodValidator: JSR-303 validator failed to initialize: Unable to create a Configuration, because no Bean Validation provider could be found. Add a provider like Hibernate Validator (RI) to your classpath. (see http://aspects.jcabi.com/jsr-303.html)
[?[2;37mDEBUG?[m] pool-1-thread-10 com.jcabi.matchers.NoBrokenLinks: #assertThat(): 1 links found: [""]
[?[2;37mDEBUG?[m] pool-1-thread-8 com.jcabi.matchers.NoBrokenLinks: #assertThat(): 1 links found: ["/broken-link"]
[?[2;37mDEBUG?[m] pool-1-thread-8 com.jcabi.matchers.NoBrokenLinks: #http('http://google.com/broken-link'): response code is 404
[?[0;33mWARN?[m] pool-1-thread-8 com.jcabi.matchers.NoBrokenLinks: #isValid('http://google.com/broken-link'): not valid since response code is 404
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 9.72 sec <<< FAILURE! - in com.jcabi.matchers.NoBrokenLinksTest
passesWithoutBrokenLinks(com.jcabi.matchers.NoBrokenLinksTest)  Time elapsed: 5.56 sec  <<< FAILURE!
java.lang.AssertionError:
Expected: 2 broken link(s) found: ["http://www.rexsl.com/index.html", "http://img.rexsl.com/logo.png"]
     but: was <200 OK [http://localhost:12345/see-FakeRequest-class]

<html xmlns='http://www.w3.org/1999/xhtml'><body><a href='/index.html'/><a href='http://img.rexsl.com/logo.png'/></body></html>>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
        at com.jcabi.matchers.NoBrokenLinksTest.passesWithoutBrokenLinks(NoBrokenLinksTest.java:91)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:318)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)

Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8

Results :

Failed tests:
  NoBrokenLinksTest.passesWithoutBrokenLinks:91
Expected: 2 broken link(s) found: ["http://www.rexsl.com/index.html", "http://img.rexsl.com/logo.png"]
     but: was <200 OK [http://localhost:12345/see-FakeRequest-class]

<html xmlns='http://www.w3.org/1999/xhtml'><body><a href='/index.html'/><a href='http://img.rexsl.com/logo.png'/></body></html>>

Tests run: 32, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

**Comment by dmarkov** [comment]
@yegor256 dispatch this issue please, see [par.21](http://at.teamed.io/policy.html#21)

**Comment by yegor256** [comment]
@simonjenga how can I reproduce this?

**Comment by simonjenga** [comment]
@yegor256 ```mvn test``` or ```mvn clean install```

**Comment by yegor256** [comment]
@simonjenga fixed in 12d5842 thanks!
